crates/spirv-builder/README.md

```diff
@@ -0,0 +1,47 @@
+<!-- inline html -->
+<!-- markdownlint-disable-file MD033 -->
+# `spirv-builder`
+
+![Rust version](https://img.shields.io/badge/rust-1.66.0_nightly--2022--10--29-purple.svg)
+
+This crate gives you `SpirvBuilder`, a tool to build shaders using [rust-gpu][rustgpu].
+
+It takes care of pulling in the `SPIR-V` backend for Rust, `rustc_codegen_spirv`, and invoking a nested build using appropriate compiler options, some of which may be set using the `SpirvBuilder` API.
+
+## Example
+
+```rust
+use spirv_builder::{MetadataPrintout, SpirvBuilder};
+
+fn main() {
+    SpirvBuilder::new("my_shaders", "spirv-unknown-vulkan1.1")
+    .print_metadata(MetadataPrintout::Full)
+    .build()
+    .unwrap(); 
+}
+```
+
+This example will build a shader crate called `my_shaders`. You typically insert this code in your crate's `build.rs` that requires the shader binary. The path to the shader module's binary will be set in the `my_shaders.spv` environment variable, which you can include in your project using something along the lines of:
+
+```rust
+const SHADER: &[u8] = include_bytes!(env!("my_shaders.spv"));
+```
+
+## Building with `spirv-builder`
+
+Because of its nature, `rustc_codegen_spirv`, and therefore `spirv-builder` by extension, require the use of a very specific nightly toolchain of Rust.
+
+**The current toolchain is: `nightly-2022-10-29`.**
+
+Toolchains for previous versions of `spirv-builder`:
+
+|Version|Toolchain|
+|-|-|
```

Comment:
```suggestion
|-:|-|
```
(to right-align the version, juts a small visual detail, heh)

Reply:
Perfect 👍